### DISCUSSION TITLE
Spaceship: Team Selection, when logging in via ConnectAPI

### DISCUSSION THREAD

**Discussion by anh1979** [fastlane/fastlane#18356] [Q&A]
Somehow the apidocs do not help me with this and I already tried several different ideas...

**I want to find about status of the app, if it is already published or still in review or rejected...**
I use
`Spaceship::ConnectAPI::App.find('com.abcdef.something')`, to find my App
that worked somehow - but not for all teams.
When I try to find an app, that is not in our first team (manually in cli), this message appears:
The current user is in xx teams. Pass a team ID or call `select_team` to choose a team. Using the first one for now.

How can I do that, when I am using the ConnectAPI login?
Before we switched to ConnectAPI, `Spaceship::Portal.select_team(team_id: ENV['FASTLANE_TEAM_ID'])` worked just fine to select that team, but in combination with the ConnectAPI login, this has no effect?!

Help really appreciated. It would also be very nice to find a bit more info about this in the docs.
Thanks in advance, Anne

**Comment by anh1979** [comment]
No idea anyone?

**Comment by dinsen** [comment]
Hi @anh1979 👋

You need to create an API key for every team you want to use Fastlane with.
The key is only for that specific team you create it for.
(It's not possible to create API keys for Enterprise accounts currently)
